acceptance/dockerImage/build/runCucumber.sh

```diff
@@ -17,6 +17,8 @@ CUCUMBER_CMD="cd /capybara; cucumber $*"
 eval ${CUCUMBER_CMD}
 EXIT=$?
 
-java -jar /usr/share/reporter/reporter.jar ./output ./output/report.json
-chown -R $CALLER_UID:$CALLER_GID ./output
-exit $EXIT
+if [ -d ./output ] && [ -f ./output/report.json ]; then
+    java -jar /usr/share/reporter/reporter.jar ./output ./output/report.json
+    chown -R $CALLER_UID:$CALLER_GID ./output
+fi
```

Comment:
To get these changes into an image, you will have to rebuild the capybara image locally with a new version number, push the new image to dockerhub, and then create a separate PR to reference that image in the run*Acceptance scripts.